pkg/exporter/proto.go

```diff
@@ -11,14 +11,21 @@ import (
 
 // flowsToPB is an auxiliary function to convert flow records, as returned by the eBPF agent,
 // into protobuf-encoded messages ready to be sent to the collector via GRPC
-func flowsToPB(inputRecords []*flow.Record) *pbflow.Records {
+func flowsToPB(inputRecords []*flow.Record, maxLen int) []*pbflow.Records {
 	entries := make([]*pbflow.Record, 0, len(inputRecords))
 	for _, record := range inputRecords {
 		entries = append(entries, flowToPB(record))
 	}
-	return &pbflow.Records{
-		Entries: entries,
+	var records []*pbflow.Records
```

Comment:
Could it be better done in one pass rather than 2 passes?

Reply:
I'll do it despite I'm afraid it will look a bit less readable. Also consider that in the second step we aren't copying arrays but just creating references to subslices pointing to the data in the original "entries" slice, so the impact in performance shouldn't be really high.
<img width="796" alt="image" src="https://user-images.githubusercontent.com/939550/210533080-aae69dc6-a40d-400b-a51d-7933ec141c68.png">
But anyway let me do the change and then decide which one do you prefer.

Reply:
no no, that's fine, you can leave it as it is
I though it involved copying slices

Reply:
thanks btw for the info!